client/src/components/CohortList/CohortList.js

```diff
@@ -10,9 +11,9 @@ import AddIcon from '@material-ui/icons/Add';
 import CohortCard from '../CohortCard/CohortCard';
 
 // function mapStateToProps(state) {
-//     return {
-//         state,
-//     };
+//   return {
```

Comment:
I think you may as well remove these comments. If we need to refer to them we can look back at git

Reply:
Sounds good I removed them from the file